.travis.yml

```diff
@@ -106,6 +110,109 @@ matrix:
               ':!*tests/*' ':!*targets/*' ':!*TARGET_*' ':!*unsupported/*' \
               ':!*events/tests/*' ':!*drivers/tests/*'
 
+    - &cmake-build-test
+      stage: "CMake"
+      name: "CMake blinky example - develop (K64F)"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-blinky TARGET_NAME=K64F PROFILE=develop
+      language: python
+      python: 3.8
+      install:
+        # Install gcc
+        - source_pkg gcc
+        - arm-none-eabi-gcc --version
+        - pip install --upgrade cmake
+        - pip install --upgrade mbed-tools
+        # We use manual clone, with depth and single branch = the fastest
+        # Because of this, we need to create .mbed file as the current tools require it
+        - git clone --depth=1 --single-branch --branch feature-cmake https://github.com/ARMmbed/${EXAMPLE_NAME}.git;
+        - >-
+          if [ -z ${SUBEXAMPLE_NAME} ]; then
+             cd ${EXAMPLE_NAME};
+           else
+             cd ${EXAMPLE_NAME}/${SUBEXAMPLE_NAME};
+           fi
+        - >-
+          git clone --depth=1 --single-branch https://github.com/ARMmbed/mbed-os.git;
+          cd mbed-os;
+        - to_fetch=("${TRAVIS_COMMIT}")
+        - if [ false != "${TRAVIS_PULL_REQUEST-}" ]; then to_fetch+=("+refs/pull/${TRAVIS_PULL_REQUEST}/merge:"); fi
+        - git fetch -q -- origin "${to_fetch[@]}"
+        - git checkout -qf "${TRAVIS_COMMIT}" --
+        - >-
+          cd -;
+        # Replace the content of mbed-os.lib to stay on ${TRAVIS_PULL_REQUEST}
+        -  echo “” > mbed-os.lib
+        -  mbedtools checkout;
+        -  mbedtools configure -t GCC_ARM -m ${TARGET_NAME};
+        -  mkdir -p build
+      script:
+        - cd build && cmake .. -GNinja -DCMAKE_BUILD_TYPE=${PROFILE} && cmake --build .
```

Comment:
```suggestion
        - mbedtools build -t GCC_ARM -m ${TARGET_NAME} -c
```

Reply:
I rebased the PR, the issue should be resolved